test/libponyc/refer.cc

```diff
@@ -838,3 +838,14 @@ TEST_F(ReferTest, ConsumeParamVarSubfieldReassignSameExpressionReuse)
   TEST_ERRORS_1(src,
     "can't use a consumed local or field in an expression");
 }
+
+TEST_F(ReferTest, ConsumeTupleAccessorOfFunctionValResult)
+{
+    const char* src = "actor Main\n"
+                      "    new create(env: Env) =>\n"
+                      "       let a = \"I am a string\"\n"
+                      "       consume a.chop(1)._1\n";
+
+    TEST_ERRORS_1(src,
+        "Consume expressions must specify a single identifier");
```

Comment:
To go along with the other change I suggested above, we would need to use lowercase here in the error message test if we are using lowercase for the error message itself.

```suggestion
      "consume expressions must specify a single identifier");
```

My suggested change also fixes indentation to use 2-space indentation rather than 4-space.

Reply:
thanks for fixing the issues. Henceforth, I will be mindful of these conventions.